flashcards/backend/docs/install-poetry.rst

```diff
@@ -0,0 +1,7 @@
+install\-poetry module
```

Comment:
this file shouldn't be documented by us, this is official poetry installation script. remove this file

Reply:
again here